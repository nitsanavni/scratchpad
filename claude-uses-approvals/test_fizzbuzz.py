from approvaltests import verify


def fizzbuzz(n):
    result = []
    for i in range(1, n + 1):
        if i % 3 == 0 and i % 5 == 0:
            result.append("FizzBuzz")
        elif i % 3 == 0:
            result.append("Fizz")
        elif i % 5 == 0:
            result.append("Buzz")
        else:
            result.append(str(i))
    return result


def test_fizzbuzz():
    verify("\n".join(fizzbuzz(15)))
