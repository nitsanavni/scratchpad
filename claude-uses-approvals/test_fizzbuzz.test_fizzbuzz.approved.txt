1
2
Fizz
4
Buzz
Fizz
Whizz
8
Fizz
Buzz
11
Fizz
13
Whizz
FizzBuzz
